### ISSUE TITLE
have no limit on crypto trades

### ISSUE THREAD

**Issue by XMRfamily** [haveno-dex/haveno#1096]
Idk why tf there is 96 xmr limit for crypto trades on haveno?  Should be no limit, for fiat trades it's understandable but limits definitely need to be raised even for signed accounts(12 xmr is small for certain payment methods)

**Comment by monerobull** [comment]
I agree that limit on crypto trades makes little to no sense. 
I think 3 XMR limit on less secure fiat payments is a good limit but the maximum for signed accounts could indeed be raised form 12.

**Comment by woodser** [comment]
I can see increasing the limit, but unlimited amounts would increase risk of being targeted by a rogue arbitrator.

**Comment by XMRfamily** [comment]
****

> I can see increasing the limit, but unlimited amounts would increase risk of being targeted by a rogue arbitrator.

Rouge arbitrator is always a risk. Signed accounts should have limited of 50/ 100 xmr and crypto-crypto should have 1000 xmr limit?

**Comment by monerobull** [comment]
I just went over the trade-limits code and it looks like it uses the 96 XMR max as input and modifies it based on a risk factor. This means this is a bigger change than just changing some hardcoded numbers :/

I'd like to see limits for signed SEPA accounts raised to the equivalent of 5-8k€ & crypto-crypto should be at least the equivalent of 1 BTC, if not more. Cash by mail could go up to 10k€ and payment apps like CashApp/Revolut/Zelle can stay at the current limits.

**Comment by XMRfamily** [comment]
> I just went over the trade-limits code and it looks like it uses the 96 XMR max as input and modifies it based on a risk factor. This means this is a bigger change than just changing some hardcoded numbers :/
> 
> I'd like to see limits for signed SEPA accounts raised to the equivalent of 5-8k€ & crypto-crypto should be at least the equivalent of 1 BTC, if not more. Cash by mail could go up to 10k€ and payment apps like CashApp/Revolut/Zelle can stay at the current limits.

I agree with everything except CashApp/Revolut/Zelle should stay the same. Limits should be raised to around $2k at least or 20 xmr
